DOCS-3352 GitLab CI/CD Integration Documentation (#14986)
DataDog/documentation: content/en/synthetics/cicd_integrations/_index.md

> {{< whatsnext desc="With Synthetics and CI/CD, you can run Synthetic tests in the CI platform provider of your choice:" >}}

Reviewer: Should we also change this to something like: 

```suggestion
{{< whatsnext desc="With Synthetics and CI/CD, you can run Synthetic tests in any CI platform provider of choice. Here are some documented integrations:" >}}
```

I'm not sure if it's great wording, but I wanted to highlight that the ability to run Synthetic tests in CI is not limited to just these providers.

Author: How about something like this?
